File: Baekjoon/수학/4153.py
# https://www.acmicpc.net/problem/4153

import sys
input = sys.stdin.readline

while True:
    a, b, c = map(int, input().split())
    if a == 0 and b == 0 and c == 0: break
    if a > b: a, b = b, a
    if b > c: b, c = c, b
    if a > b: a, b = b, a
    print('right' if a ** 2 + b ** 2 == c ** 2 else 'wrong')

File: Baekjoon/수학/4153.cpp
// https://www.acmicpc.net/problem/4153

#include <iostream>
#include <algorithm>
using namespace std;

int main() {
    while (true) {
        int a, b, c;
        cin >> a >> b >> c;
        if (a == 0 && b == 0 && c == 0) break;
        if (a > b) swap(a, b);
        if (b > c) swap(b, c);
        if (a * a + b * b == c * c) {
            cout << "right\n";
        } else {
            cout << "wrong\n";
        }
    }
    return 0;
}
